ext/ddtrace_profiling_native_extension/native_extension_helpers.rb

```diff
@@ -88,7 +88,7 @@ def self.unsupported_reason
             on_macos? ||
             on_unknown_os? ||
             not_on_amd64_or_arm64? ||
-            on_ruby_2_1? ||
+            on_ruby_2_1_or_2_2? ||
```

Comment:
Minor: sounds unwieldy already, maybe change that to either one of:

- `on_ruby_2_1? || on_ruby_2_2?`
- `on_ruby?('2.1') || on_ruby?('2.2')`
- `on_unsupported_ruby_version?`

or something to that effect.

Reply:
Fixed in ff8eeadd91cfb91d0b57504ac33b81da8d4f7e1c